cli/command/system/testdata/docker-info-with-swarm.golden

```diff
@@ -21,6 +21,7 @@ Swarm: active
  ClusterID: 9vs5ygs0gguyyec4iqf2314c0
  Managers: 1
  Nodes: 1
+ Data Path Port: 0
```

Comment:
With a daemon that supports this new feature, this would show `4789` (correct?) if so, can you change this to `4789`?

Reply:
if I change to 4789 then CI fails now. If that is ok with you , I can go ahead and change it to 4789. To keep CI green, I made it to 0 as of now.  Let me know how do you like to proceed?

Reply:
It's failing in that case, because the `DataPathPort` isn't set in that test; the test is using the info that's set in https://github.com/docker/cli/blob/ca782599fbf5bcea70b12fc05fc1b35cb8bebc64/cli/command/system/info_test.go#L125-L191

See https://github.com/docker/cli/blob/ca782599fbf5bcea70b12fc05fc1b35cb8bebc64/cli/command/system/info_test.go#L194-L195


If we want to test the output with and without `DataPathPort` set, we could add two test-cases (and corresponding `.golden` files) in https://github.com/docker/cli/blob/ca782599fbf5bcea70b12fc05fc1b35cb8bebc64/cli/command/system/info_test.go#L231-L251

IMO that's a "nice to have", but not super-critical (the `docker info` part is quite trivial)